src/js/ToolTip.js

```diff
@@ -1,38 +1,60 @@
-/*global module*/
-
-function ToolTip(text, refEl) {
-	"use strict";
-
-	var tipEl;
-	var cssClass = "o-share-tooltip";
-
-	function create() {
-		tipEl = document.createElement('div');
+/**global module*/
+'use strict';
+
+/**
+ * @class Tooltip
+ *
+ * @param {string} text
+ * @param {HTMLElement} refEl
+ */
+function Tooltip(text, refEl) {
+	this.cssClass = 'o-share-tooltip';
+
+	/**
+	 * Creates a tooltip element
+	 *
+	 * @private
+	 * @returns {HTMLElement}
+	 */
+	function createTooltip(cssClass) {
+		var tipEl = document.createElement('div');
 		tipEl.className = cssClass;
 		tipEl.innerHTML = '<div class="' + cssClass + '__text">' + text + '</div><div class="' + cssClass + '__arrow"></div>';
-		refEl.insertBefore(tipEl, refEl.childNodes[0]);
-		tipEl.style.width = tipEl.clientWidth + "px"; // Set width based on initial text
+		return tipEl;
 	}
 
-	function position() {
+	/**
+	 * Renders a tooltip element
+	 *
+	 * @private
+	 * @param {HTMLElement} tipEl - a tooltip element returned by {@link createTooltip}
+	*/
```

Comment:
Not sure it has an impact on the JSDoc parser but there should be one more space before `*`